src/goto-symex/symex_assign.cpp

```diff
@@ -215,14 +215,6 @@ void symex_assignt::assign_non_struct_symbol(
       assignment.rhs,
       target,
       symex_config.allow_pointer_unsoundness);
-    // Erase the composite symbol from our working state. Note that we need to
```

Comment:
There could well be a good justification for this removal of code but please could they not be in a documentation PR?